recipes/mammoth/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "mammoth" %}
```

Comment:
I believe this is the same as https://github.com/conda-forge/staged-recipes/pull/28583

@sarthakpati  do you want to be added as a maintainer here so we can merge this one?

Reply:
Sure thing, thanks!

Reply:
done in 7222a1518623f30af21df47ed2f8d6abeffe23cc ✅